proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+# Cosmos Stargate Hub Upgrade Proposal 2: Chevrons Locked. Time to Upgrade
+
+![Stargate Upgrade banner graphic](https://siasky.net/AADU5rg0GvapbrICLUk7SxZeMDrKo5Q8oLrpOwl71wXhnQ)
+
+## by Iqlusion
+
+Jan 11, 2021
```

Comment:
kind of a nit, but do you want to update the file names for both these files to be `2021-01` rather than `2020-11`?